public/docs/ts/latest/quickstart.jade

```diff
@@ -46,8 +41,8 @@
     are loaded, or configure your editor or IDE to do it.
 
   code-example.
-    $ npm install -g typescript@^1.5.0-beta
-    $ tsc --watch -m commonjs -t es5 --emitDecoratorMetadata app.ts
+    $ tsc --init --experimentalDecorators --target es5 --emitDecoratorMetadata
```

Comment:
Ignore QuickStart ... it is dead ... soon to be replaced by Getting Started.

We'll remove QS very soon and replace with GS.

Alex may not know about this plan yet.